src/Microsoft.DotNet.Wpf/src/PresentationFramework/MS/Internal/Controls/WebBrowserEvent.cs

```diff
@@ -248,14 +248,23 @@ public void DocumentComplete(object pDisp, ref object url)
             }
         }
 
+        // if we are navigating to "about:blank", ignore completion events
+        // for any other url.  These can happen despite our attempt to cancel
+        // outstanding navigations (see WebBrowser.DoNavigate), due to race
+        // conditions between the browser and us.
+        private bool ShouldIgnoreCompletionEvent(ref object url)
```

Comment:
Do we really need ref here?

Reply:
Maybe not, but copying the conventions of existing code.  There's no gain in changing it.